packages/ui/tsconfig.json

```diff
@@ -7,7 +7,7 @@
       "dom.iterable",
       "esnext"
     ],
-    "allowJs": true,
+    "allowJs": false,
```

Comment:
ooh, strict!

Reply:
The dev server is slow right now coz of typescript, this speeded up a little. Also we don't have to use any separate js file right, it can be ts with js too. @sanathkr